build/arch.mips.mk

```diff
@@ -24,4 +24,9 @@ endif
 # Added to all files
 GCC_ABIFLAGS += -msoft-float
 CLANG_ABIFLAGS += -msoft-float
+ifeq ($(KGPROF), 1)
+  CFLAGS_KGPROF = -finstrument-functions \
+                  -finstrument-functions-exclude-file-list=spinlock.c  \
+                  -finstrument-functions-exclude-function-list=intr_disable,intr_enable
```

Comment:
Why you don't use `__no_instrument_function` for these functions?

Reply:
I left it for discussion cause:
1. `intr_disable` and  `intr_enable` are the only substrings, which cover more than one function definition
2. I am not sure if we want to use `__no_instrument_function` only for `copystr` otherwise, if we would add in the near future another program, which will use `finstrument-functions` there is a high probability we will move the definitions back to `-finstrument-functions-exclude-function-list=`.